resources/testdata/glyphs2/ProductionNames.glyphs

```diff
@@ -0,0 +1,56 @@
+{
+.appVersion = "3413";
+familyName = "Production Names";
+fontMaster = (
+{
+alignmentZones = (
```

Comment:
you could delete `alignmentZones` and all the metrics and this would read a bit more cleanly.